environments/docker.toml

```diff
@@ -31,6 +31,7 @@
         "log_value",
         "pot_cage",
         "pot_file_dsr",
+        "pot_file_vow",
```

Comment:
What's this got to do with this PR? :thinking:

Reply:
I think this PR just needs a rebase - these pot_file_vow changes were merged into staging already